pkg/bot/mattermost_fmt.go

```diff
@@ -0,0 +1,97 @@
+package bot
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/kubeshop/botkube/pkg/events"
+	format2 "github.com/kubeshop/botkube/pkg/format"
```

Comment:
```suggestion
	formatx "github.com/kubeshop/botkube/pkg/format"
```
x - for extension

please apply it for other places too, thanks!